src/main/java/mate/academy/hibernate/relations/dao/impl/CountryDaoImpl.java

```diff
@@ -12,11 +15,33 @@
 
     @Override
     public Country add(Country country) {
-        return null;
+        Session session = null;
```

Comment:
Consider using try-with-resources for the `Session` to ensure it is closed automatically, which can simplify the code and reduce the chance of resource leaks.